### DISCUSSION TITLE
Column wrapping in Data Grid?

### DISCUSSION THREAD

**Discussion by apombo-ux** [twilio-labs/paste#2185] [Q&A] [answered]
I'm looking at using a Data Grid rather than a Table to show a list of Studio Flows with actions users can take. Data Grid seems like the right choice because I'll have actionable elements, but it looks like the Data Grid cells resize (get wider) to fit everything in them. In the case of, say, a long friendly name for a flow, those actionable elements can end up hidden behind a horizontal scroll.

Does the Data Grid have any column wrapping like the Table does, or have other ways to manage the overall component width (cell truncation, or responsive column resizing between a min and max column width) so it can be kept at 100% of the page?

Edited to clarify that my goal is to not have horizontal scrolling in this data grid. The [composition notes for Table](https://paste.twilio.design/components/table/#composition-notes) say "Tables fill 100% width of the space of their containing element"- that's ultimately what I'm wanting here. Is that possible with the Data Grid component?

Here's an example of the design I'm working with: 
![image](https://user-images.githubusercontent.com/98772241/152610781-d930d7b2-cc72-427e-80a5-b4f38e5b16c3.png)

**Comment by gloriliale** [comment, marked as answer]
Hi @apombo-ux! We're working on some possible solutions from our end, and for ease of comms, I'll be updating the Office Hours thread ([Discussion #2190](https://github.com/twilio-labs/paste/discussions/2190)) about the issue. Thanks for posting!
